CHANGELOG.md

```diff
@@ -8,9 +8,59 @@ and this project adheres to [Semantic Versioning].
 
 ## [0.9.0] — unreleased
 
+In this release we have been focused on improving performance of the Baseline interpreter.
+The end result is that the Baseline is **26% faster** than in previous version 0.8.0
+and **18% faster** than the Advanced interpreter while having 
+over **8x smaller** code analysis cost. The Baseline is now the _default_ interpreter because 
+it is simpler and has become better than the Advanced. 
+
+The Advanced also has got **4% faster** than in the previous version.
+
+All numbers are from running the "main" benchmark suite
+on 4.0 GHz Intel Haswell CPU, using the Clang 15 compiler.
+
```

Comment:
Would also highlight the gas refunds part here.

Reply:
Added.